config/settings.yml

```diff
@@ -32,6 +32,13 @@ discourse_solved:
     client: true
   notify_on_staff_accept_solved:
     default: false
+  prevent_assign_notification:
+    default: false
+  assign_to_status_on_solved:
+    default: false
+  status_to_assign_on_solved:
```

Comment:
`assignment_status_on_solve` maybe? Since it's not the status _to_ assign, it's the status that assignments get updated to 🤷